### ISSUE TITLE
With a clean installation of the application: "Safari detected an app or service that interfered with clicking"

### ISSUE THREAD

**Issue by Oleg-Chashko** [AdguardTeam/AdGuardForSafari#594]
- AdGuard Safari v.1.9.14 Beta
- Safari v.14.1 (16611.1.21.161.6)
- Big Sur v.11.3.1 (20E241)

https://user-images.githubusercontent.com/62497891/117542385-b9edb880-b018-11eb-9ad5-d7d4f9fcd88a.mp4

**Comment by Oleg-Chashko** [comment]
As I described above, this behavior does not manifest itself in every installation. I am checking the application more deeply! I install the application, then delete the application, with a complete cleanup of the remains of the application. And so many times until I see the problem.

**Comment by Oleg-Chashko** [comment]
Moreover, this behavior is only in AdGuard Safari v.1.9.14 Beta. There are no problems with AdGuard Safari v.1.9.13!

**Comment by zzebrum** [comment]
@Oleg-Chashko it is a system notification, which appears not only with our extension. And not only on v1.9.14 :) 
https://support.apple.com/en-us/HT211233

**Comment by zzebrum** [comment]
@Oleg-Chashko indeed, it happens not every time and only Apple guys can say why. But I can assure you that it has nothing to do with the update, because it has happened before with previous versions too.
